src/lib/openapi/spec/admin-features-query-schema.ts

```diff
@@ -0,0 +1,26 @@
+import { FromSchema } from 'json-schema-to-ts';
+
+export const adminFeaturesQuerySchema = {
+    $id: '#/components/schemas/adminFeaturesQuerySchema',
+    type: 'object',
+    additionalProperties: true,
+    properties: {
+        tag: {
+            type: 'array',
+            items: {
+                type: 'array',
+                items: {
+                    type: 'string',
+                },
+            },
+        },
+        namePrefix: {
+            type: 'string',
```

Comment:
What does this query param do? 

```suggestion
            type: 'string',
            description: '',
```